lib/monitor/monitor.go

```diff
@@ -76,14 +73,12 @@ func NewMonitor(stats *Stats) *Monitor {
 // It needs the stats struct pointer to update when measures come
 // Return an error if something went wrong during the connection setup
 func (m *Monitor) Listen() error {
-	ln, err := net.Listen("tcp", Sink+":"+strconv.Itoa(m.SinkPort))
+	ln, err := net.Listen("tcp", Sink+":"+strconv.Itoa(DefaultSinkPort))
 	if err != nil {
```

Comment:
I know it's just I took the work I've been working on 3 -4 weeks ago so your changes were not included yet. I'm working on this (also related to passing the race tests)